apollo-api/api/apollo-api.api

```diff
@@ -17,6 +17,9 @@ public final class com/apollographql/apollo3/api/AdapterContext$Builder {
 	public final fun variables (Lcom/apollographql/apollo3/api/Executable$Variables;)Lcom/apollographql/apollo3/api/AdapterContext$Builder;
 }
 
+public final class com/apollographql/apollo3/api/AdapterContextKt {
+}
```

Comment:
Interesting that this is adding a public class given that all its symbols should be internal IIUC. Maybe rename to `-AdapterContext.kt` to make sure it's never reachable in Java ?